datacalls.go

```diff
@@ -10,6 +10,19 @@ const (
 	_DATA_PREAMBLE = "/api/v/1/data/"
 )
 
+type Collection struct {
+	SystemKey    string
+	CollectionID string
+	Name         string
+	Columns      []Column
+	Users        bool
+}
+
+type Column struct {
+	name       string
```

Comment:
Any reason why ColumnType is exported and name isn't? also naming feels inconsistent here @sh3nan1gans @aallsbrook